### ISSUE TITLE
Scheduled delivery image/csv jobs should only run once 

### ISSUE THREAD

**Issue by rephus** [lightdash/lightdash#4595]
### Description

Right now we run different full jobs per target, which means a dashboard scheduled image delivery can make several requests to headless browser to fetch the same screenshot. 

![image](https://user-images.githubusercontent.com/1983672/221186743-9e5a6ae9-4065-476a-9d62-d452095e427a.png)

I propose the following change:

- Run 1 task per scheduled job
- This job will fetch the info from the chart/dashboard + image or CSV
- Then create  separate jobs per target with the generated image or CSV

**Comment by ZeRego** [comment]
Yup, the only con of that approach is that it can result in partial success. 🤔 
So on each retry, we might send duplicate slack messages/emails
